docs/services/navigation/_index.md

```diff
@@ -564,7 +580,15 @@ obstacles = await my_nav.get_obstacles()
 {{% /tab %}}
 {{< /tabs >}}
 
-## Concepts
+## Control Tab Usage
+
+After configuring the navigation service for your robot, navigate to the **Control** tab of the robot's page in the [Viam app](https://app.viam.com) and expand the card matching the name of your service to utilize an interactive portal for rover navigation.
```

Comment:
Prefer simpler language 
```suggestion
After configuring the navigation service for your robot, navigate to the **Control** tab of the robot's page in the [Viam app](https://app.viam.com) and expand the card matching the name of your service to use an interactive portal for rover navigation.
```

Reply:
I don't know how I feel about interactive portal. Is that what it's called?

Reply:
I'm fine changing it but I don't really know what to say. Maybe `interface`?